Essentials/src/messages.properties

```diff
@@ -413,7 +413,7 @@ pTimeSetFixed=\u00a76Player time is fixed to \u00a7c{0}\u00a76 for\: \u00a7c{1}.
 pWeatherCurrent=\u00a7c{0}\u00a76''s weather is\u00a7c {1}\u00a76.
 pWeatherInvalidAlias=\u00a74Invalid weather type
 pWeatherNormal=\u00a7c{0}\u00a76''s weather is normal and matches the server.
-pWeatherOthersPermission=\u00a74You are not authorized to set other players weather.
```

Comment:
This is still incorrect. Revert back to `players''`, as it is the correct grammar.